### DISCUSSION TITLE
error but chart still run?

### DISCUSSION THREAD

**Discussion by nqnga** [highfestiva/finplot#339] [Q&A]
Can I ask after I updated new version, I had this issue:
Traceback (most recent call last):
  File "D:\astro\venv\lib\site-packages\finplot\__init__.py", line 2647, in _x2t
    s = ts2str(t)
  File "D:\astro\venv\lib\site-packages\finplot\__init__.py", line 2630, in <lambda>
    lambda t: _millisecond_tz_wrap(datetime.fromtimestamp(t / 1e9, tz=display_timezone).isoformat(sep=' ')))
OSError: [Errno 22] Invalid argument

chart showed but have no time
![image](https://user-images.githubusercontent.com/9681519/172354876-a7c3a457-b39e-41b6-b092-7fe86b7fc669.png)

how can I fix it?
Thanks

**Comment by highfestiva** [comment]
`datetime.fromtimestamp()` can only convert timestamps after epoch. So try removing all dates before 1970.

**Reply by nqnga** [reply]
how can I get the old version of your app, last time I can plot it?

**Reply by highfestiva** [reply]
https://stackoverflow.com/questions/5226311/installing-specific-package-version-with-pip

**Reply by nqnga** [reply]
I forgot what version can plot data before 1970 :) I tried several but cant not run. Dont know why...

**Reply by highfestiva** [reply]
I forgot too. Let me know if you remember!
